frontend/src/components/TransactionsPerSecond/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { useEffect, useMemo } from "react";
+import { useDispatch } from "react-redux";
+import BigNumber from "bignumber.js";
+
+import { useRedux } from "hooks/useRedux";
+import {
+  ActionStatus,
+  LedgerTransactionHistoryFilterType,
+  Network,
+} from "types";
+import { SectionCard } from "components/SectionCard";
+import {
+  VerticalBarChart,
+  TimeRange,
+} from "components/charts/VerticalBarChart";
+import { fetchTransactionsHistoryAction } from "ducks/transactions";
+import "./styles.scss";
+import { AmountInfoCard } from "components/AmountInfoCard";
+import { Icon } from "@stellar/design-system";
+
+export const TransactionsPerSecond = () => {
+  const { transactions, ledgers } = useRedux("transactions", "ledgers");
+  const dispatch = useDispatch();
+  const lastLedger = ledgers.lastLedgerRecords[0];
+  const selectedTimeInterval = LedgerTransactionHistoryFilterType["30D"];
+
+  useEffect(() => {
+    dispatch(fetchTransactionsHistoryAction(Network.MAINNET));
+  }, [dispatch, selectedTimeInterval]);
+
+  const data = useMemo(() => {
+    const result = transactions.transactionsHistory.items.map((item) => ({
+      date: new Date(item.date),
+      primaryValue: new BigNumber(
+        new BigNumber(item.txTransactionCount / 84600).toFormat(2),
+      ).toNumber(),
+      secondaryValue: 0,
```

Comment:
I'm not very familiar with `VerticalBarChart` but why do we need `secondaryValue` here? Since we're using `primaryValueOnly`, is it needed?

Reply:
we don’t need this field, he was like this because before this was a required field